src/routerd_lib/main.cpp

```diff
@@ -266,7 +309,8 @@ namespace NAC {
             args.BindPort6 = args.BindPort4 = config["port"].get<unsigned short>();
             args.ThreadCount = ((config.count("threads") > 0) ? config["threads"].get<size_t>() : 10);
             args.ClientArgsFactory = [&router, &requestFactory]() {
-                return new NHTTPServer::TClient::TArgs(router, std::forward<NHTTPServer::TClient::TArgs::TRequestFactory>(requestFactory));
+                return new NHTTPServer::TClient::TArgs(
+                        router, std::forward<NHTTPServer::TClient::TArgs::TRequestFactory>(requestFactory));
```

Comment:
Please do not change formatting of existing code just for the sake of changing formatting.